distribution/ddf-common/src/main/resources/bin/ddf.bat

```diff
@@ -8,58 +8,21 @@ SET DDF_HOME=%CD%
 POPD
 
 SET GET_PROPERTY=%DIRNAME%get_property.bat
-SET SOLR_EXEC=%DDF_HOME%\solr\bin\solr.cmd
+SET SOLR_EXEC=%DDF_HOME%\bin\ddfsolr.bat
 
-:restart
+:RESTART
 REM Remove the restart file indicator so we can detect later if restart was requested
 IF EXIST "%DIRNAME%restart.jvm" DEL "%DIRNAME%restart.jvm"
 
 REM Get Solr start property
-CALL %GET_PROPERTY% start.solr true
+CALL %GET_PROPERTY% start.solr
```

Comment:
Calls to the get_propetry.bat should have a second argument that provides a default return value in the event that a property can't be found.

Reply:
This is currently `solr.start` in the `system.properties`. We can update either.

Reply:
Or we just hardcode the value in the script and not look system.properties.